### DISCUSSION TITLE
What are the options for the "PistonMovement" tag in config.yml?

### DISCUSSION THREAD

**Discussion by sargon64** [GriefPrevention/GriefPrevention#1114] [Q&A] [answered]
This tag is not in the wiki (to my knowledge). 
I have tried "true" "false" and "ALL". When it is set to these, it defaults to "CLAIMS_ONLY" on server startup. The question is this: What are the options for this option?

**Comment by Jikoo** [comment, marked as answer]
https://github.com/TechFortress/GriefPrevention/wiki/Setup-and-Configuration#pistonmovement
